src/fileCollectionFromFileList.ts

```diff
@@ -0,0 +1,22 @@
+import { FileCollection } from './FileCollection';
+
+/**
+ * Generate a FileCollection from a directory path
+ * By default this method will expand all zip and gzip files
+ * @param path
+ * @returns
+ */
+export async function fileCollectionFromFileList(
+  fileList: FileList,
+): Promise<FileCollection> {
+  //@ts-expect-error All implementation of FileList are done as an array
+  return fileList.map((file: File) => ({
+    name: file.name,
+    size: file.size,
+    relativePath: file.webkitRelativePath,
+    lastModified: file.lastModified,
+    text: file.text,
+    arrayBuffer: file.arrayBuffer,
+    stream: file.stream,
```

Comment:
@targos 

We have typescript error at this level (Avoid referencing unbound methods which may cause unintentional scoping of `this`) and I don't know how to solve this.

Not sure I may simply copy those properties in fact.

Reply:
you cannot "simply" copy methods. you need to do something like `stream() { return file.stream() }`

Reply:
or: `stream: file.stream.bind(file)`